### PR TITLE
Fixes #133: Load up to 10 previous ayahs

### LINKED ISSUE #133: Going to an exact ayah via hyperlink does not allow scrolling up
1. Go to http://quran.com/41/32
2. Ayah 41:32 should load as the first ayah on the screen (with the bismillah above it)

The page lets you scroll down to see latter ayahs, but you can't scroll up to see the prior ones.

screen shot 2015-10-09 at 22 32 19

### PR DESCRIPTION
**Pull request by reshadn**
- Also adjust url path

### CODE REVIEW

**Comment by mmahalwy** [comment]
lgtm, how will you handle if someone goes to /2/1 and wants to click 'Load More' on the top ;)

File: src/scripts/routes/Surah.js [hunk 1]
```diff
@@ -86,22 +87,34 @@ class Surah extends React.Component {
...
-      to: currentAyah + 10
+      to: toAyah
     });
-  }
 
-  renderLoadMore() {
+    this.context.executeAction(navigateAction, {
+      url: '/' + surahId + '/' + currentAyah + '-' + toAyah
+    });
+}
```
Comment (mmahalwy) [inline src/scripts/routes/Surah.js]: tests failing cause of this alignment

**Comment by reshadn** [comment]
Good call on the 2/1 case. I was debating on how much additional logic I should add. But I completely missed this use case. I'll remove the load more button when the user is on the first ayah.

**Comment by reshadn** [comment]
@mmahalwy Also curious how you were able to find that formatting error? I didn't find anything in the ci build results: https://travis-ci.org/quran/quran.com-frontend/builds/88500194

**Comment by reshadn** [comment]
Ok found the error logs in CI. Will get my editor to default 2 spaces more consistently.

**Comment by mmahalwy** [comment]
lgtm! Let's squash and merge :)

**Pull request merged**